login-frontend/src/App.tsx

```diff
@@ -87,32 +85,30 @@ export function App() {
 
   const handleSIOPV2Login = useCallback(
     async (nonce: string, event: FormEvent<HTMLFormElement>) => {
-
-      connect(process.env.DATABASE_URL ? process.env.DATABASE_URL : "wss://peregrine.kilt.io")
+      connect(process.env.DATABASE_URL ? process.env.DATABASE_URL : 'wss://peregrine.kilt.io');
```

Comment:
Strange env Name for a fullnode

Reply:
[3748bb2](https://github.com/KILTprotocol/opendid/pull/28/commits/3748bb284a53dc5dc989ae130bf5bb42adbf01a6)